.golangci.yml

```diff
@@ -167,11 +167,6 @@ issues:
       path: cmd/envtool
       text: pgdb
 
-    # only `pg` handler can import `pjson` package, no other handler can do that
```

Comment:
In principle, we can use the same rule for `sjon`: only `pg` and future `sqlite` handlers will use `sjson`. For now, we can keep the rule for `pg`, and when `sqlite` handler is ready, we can add a similar rule for it.

See also line 15 in this file, that's one more place mentioning `pjson`.

Reply:
Good point. Updated! Thanks. Also fixed the reference to `pjson`.